### DISCUSSION TITLE
Is there support for multiple displays?

### DISCUSSION THREAD

**Discussion by maximredmond** [sickcodes/Docker-OSX#444] [Q&A]
Is there any way to configure this to support 2 or more displays?

**Comment by sickcodes** [comment]
I don't believe so, https://github.com/sickcodes/Docker-OSX/issues/78

However, @iambenmitchell might be able to answer this using the remote display?

**Comment by iambenmitchell** [comment]
There’s 3 ways I know 

1. Pass through a AMD GPU into the VM, then use the outputs normally. (Best option) 
2. Qemu supports multiple displays. I’m not sure how to do it though. 
3. Run software on the mac that makes a virtual display. Again i’m not sure of any software that does this
